### ISSUE TITLE
Example project / Demo implementation

### ISSUE THREAD

**Issue by ludgerey** [conceptadev/typeorm-seeding#1]
Could you provide an example project how to get this running and maybe provide some best practices that work in your eyes?

For me it's kind of an engineering task to fit all the peaces together. For example did you introduce the concept of a Seeding Source, which adds another level of configuration that one needs to understand.

I think having a demo code that definitely works could help a lot.

**Comment by MrMaz** [comment]
@ludgerey absolutely! it might take a bit, so is there anything i can help with to unblock you in the short term?

**Comment by ludgerey** [comment]
I'm moving from `w3tecch/typeorm-seeding` to this project. On top of that I'm updating to typeorm v0.3.0, which adds some complexity, too. However migrating the Seed and Factory classes was straightforward.

But Seeding Sources adds a new concept, which I need to understand. To be fair I have to read the documentation more carefully. I skimmed trough a bit. Shame on me.

Furthermore configuration ormconfig.ts and DataSource did change, which I probably have to migrate too.

All this combined makes it hard to understand what should be the final result of the migration. What will work.

**Comment by MrMaz** [comment]
@ludgerey we have it implemented on this starter project for our Rockets library. The seeders and factories are imported from other modules, but at least you can see a working example of the orm and seeder configs with TypeORM 0.3

https://github.com/conceptadev/rockets-starter/tree/main/packages/rockets-api/src

**Comment by ludgerey** [comment]
Thanks for providing the repository! This helped me a lot.
